plugins/grpc/handlers.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package stats
+
+import (
+	"context"
```

Comment:
gRPC-go still imports `"golang.org/x/net/context"` (https://github.com/grpc/grpc-go/issues/711).
It will be a massive incompatible API change for all users using <=go1.8 if we change that.

go treats `func("context".Context)` and `func("golang.org/x/net/context".Context)` as two different types, which means the stats handler here won't work (it doesn't compile, with <=go1.8).

go1.9 introduced a new feature called type alias, which makes `"golang.org/x/net/context"` and `"context"` the same type. So, this actually works fine with >=go1.9.

The solution I can think of right now is to import `"golang.org/x/net/context"` in the stats package.

Or, if we decide that the stats package only supports users with >=go1.9, the current code will be fine.

Reply:
Since this is our first release, I think we should avoid creating unnecessary confusion and say that we support >= go1.9 if using stats with grpc.
Wondering if @rakyll have a different opinion.